lib/MusicBrainz/Server/Entity/URL/LyricWiki.pm

```diff
@@ -5,6 +5,13 @@ use Moose;
 extends 'MusicBrainz::Server::Entity::URL';
 with 'MusicBrainz::Server::Entity::URL::Sidebar';
 
+override href_url => sub {
+    # Turn the official permalink into what LyricWiki currently redirects to.
+    # See https://community.fandom.com/wiki/Help:Fandom_domain_migration for more info
+    shift->url->as_string =~
+        s{^https?://lyrics\.wikia\.com/}{https://lyrics.fandom.com/}r;
+};
+
 sub sidebar_name { 'LyricWiki' }
 
 sub url_is_scheme_independent { 1 }
```

Comment:
That should be removed as `href_url` is overridden for LyricWiki too, see https://github.com/metabrainz/musicbrainz-server/pull/1021#discussion_r278022541